.github/workflows/build.yml

```diff
@@ -0,0 +1,66 @@
+name: Quality Assurance
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+#  Enable once we have unit tests
+#  phpunit:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - uses: actions/checkout@v2
+#      - uses: shivammathur/setup-php@v2
+#        with:
+#          php-version: 7.2
+#          coverage: none
+#      - id: composer-cache
+#        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+#      - uses: actions/cache@v2
+#        with:
+#          path: ${{ steps.composer-cache.outputs.dir }}
+#          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+#          restore-keys: |
+#            ${{ runner.os }}-composer-
+#      - run: composer install --no-interaction
+#      - run: vendor/bin/phpunit
```

Comment:
Am I missing something? If there are no tests this will pass.

Reply:
Hmmm, I've added this before, but it always failed when I had no tests. I can try it again, maybe it was a config or something.